app/controllers/application_controller.rb

```diff
@@ -5,6 +5,11 @@ class ApplicationController < ActionController::Base
   include SessionsHelper
 
 #Including some general functions here so they can be reached by several controllers
-
+	def require_user
+	  if !logged_in?
+	    flash[:danger] = "You must be logged in to perform that action"
+	    redirect_to root_path
+	  end
+	end
```

Comment:
1) Include a brief comment about what the function does.
2) There is no reason to have this function here.  It is related to a session and is using another session function so it belongs inside the SessionsHelper.  Since the helper is included in application_controller all those functions are available to every controller anyway